### PR TITLE
Fix missing channelID

### PR DESCRIPTION
**Pull request by carsten19**
Fix display of network map when no channelID is present

### CODE REVIEW

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Please use `const` (or `let`) when adding/changing functionality, not `var`.
Looks good otherwise.

**Comment by carsten19** [comment]
done

File: main.js [hunk 1]
```diff
@@ -201,15 +201,15 @@ function startAdapter(options) {
...
-                                var neighbors = zwave.getNodeNeighbors(nodeID);
+                                const neighbors = zwave.getNodeNeighbors(nodeID);
                                 for (var n in neighbors) {
                                     if (!edges[neighbors[n]]) {
                                         edges[nodeID].push(neighbors[n]);
                                     }
                                 }
-                                var channelID = calcName(nodeID);
+                                const channelID = calcName(nodeID);
                                 let label = objects[channelID] ? objects[channelID].common.name : "Node" + nodeID;
```
Comment (AlCalzone) [inline main.js]: label is never changed => `const`

File: main.js [hunk 2]
```diff
@@ -201,15 +201,15 @@ function startAdapter(options) {
...
-                                var item = {
+                                let item = {
```
Comment (AlCalzone) [inline main.js]: item is also never changed ;)

**Review by AlCalzone — CHANGES_REQUESTED** [review]
sorry :)

File: main.js (hunk 1 above)
Comment (carsten19) [reply main.js]: Alright ;)

File: main.js (hunk 2 above)
Comment (carsten19) [reply main.js]: OK OK :)

**Comment by carsten19** [comment]
Done round 2 :)

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**